cypress/tests/inventory-card.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { DATA_FEDERATION_NAMESPACE, MINUTE, SECOND } from '../constants/common';
+import {
+  DATA_SOURCE_INPUTS,
+  Providers,
+  secretInputs,
+  SINGLE_BUCKET_POLICY,
+  TEST_DATA_SOURCE,
+} from '../constants/tests';
+import { BPCommon } from '../views/bucket-policies';
+import { app } from '../views/common';
+import { DSCommon } from '../views/data-resource';
+import { MCGMSCommon } from '../views/mcg-ms-common';
+
+describe('admin dashboard - inventory card', () => {
+  const PROCESSING_DS = 'e2e-processing-data-source';
+
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    MCGMSCommon.visitMcgMsDashboard();
+    app.waitForLoad();
+  });
+
+  after(() => {
+    MCGMSCommon.visitMcgMsDashboard();
+    MCGMSCommon.visitBucketPolicyList();
+    BPCommon.deleteFromListPage(SINGLE_BUCKET_POLICY);
+    MCGMSCommon.visitDataSourceListPage();
+    DSCommon.deleteFromListPage(PROCESSING_DS);
+    cy.wait(30 * SECOND);
+    DSCommon.deleteFromListPage(TEST_DATA_SOURCE);
+    // to make sure no resource are hanging around
+    DSCommon.deleteFromCmd(PROCESSING_DS, DATA_FEDERATION_NAMESPACE);
+    DSCommon.deleteFromCmd(TEST_DATA_SOURCE, DATA_FEDERATION_NAMESPACE);
+    cy.byTestID('resource-row', { timeout: 2 * MINUTE }).should('not.exist');
+  });
```

Comment:
Following this [recommendation](https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks), can this logic be moved to `before` hook (after login), so each test can start on a clean state? 

We should think about ways to mitigate the performance penalty as it seems that just this cleanup can take itself more than 2 minutes...